src/index.js

```diff
@@ -61,7 +62,7 @@ export default function (CustomElement, opts) {
       });
     }
     render() {
-      return React.createElement(tagName, null, this.props.children);
+      return React.createElement(tagName, { style: this.props.style }, this.props.children);
```

Comment:
I mean here instead of 
```js
return React.createElement(tagName, { style: this.props.style }, this.props.children);
```

Is there any downside to just simply
```js
return React.createElement(tagName, { this.props }, this.props.children);
```